server/testing_service.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	pb "github.com/googleapis/gapic-showcase/server/genproto"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// NewTestingServer returns a new TestingServer for the Showcase API.
+func NewTestingServer(observerRegistry GrpcObserverRegistry) pb.TestingServer {
+	name := fmt.Sprintf("sessions/-")
+	defaultSession := NewSession(name, pb.Session_V1_LATEST, observerRegistry)
+	sessions := []sessionEntry{sessionEntry{session: defaultSession}}
```

Comment:
Nit: You can omit the struct field name when in-lining new structs & values. Especially when we are including only 1 of 2 fields (and here it seems a little repetitive).

I'm not 100% sure on the convention, but this is just a small nit.

Reply:
Resolving due to offline conversation.